Makefile

```diff
@@ -42,9 +39,10 @@ build-image: dist
 	cp -R @jkcfg build/image/
 	docker build -t jkcfg/kubernetes -f Dockerfile build/image
 
-${SCHEMA_ZIP}:
+${SCHEMA_DIR}:
 	mkdir build
```

Comment:
This should probably have been `mkdir -p build`

Reply:
yup